config/go-app-ussd_higherhealth_healthcheck.afr_ZA.json

```diff
@@ -0,0 +1,263 @@
+{
+    "": {
+        "pot-creation-date": "2020-07-22 08:53:+0000",
+        "project-id-version": "",
+        "language-team": "",
+        "mime-version": "1.0",
+        "content-type": "text/plain; charset=UTF-8",
+        "content-transfer-encoding": "8bit",
+        "po-revision-date": "2020-07-23 11:39+0200",
+        "x-generator": "Poedit 2.3.1",
+        "plural-forms": "nplurals=2; plural=(n != 1);",
+        "last-translator": "",
+        "language": "afr"
+    },
+    "Welcome back to HealthCheck\n\nReply": [
+        null,
+        ""
+    ],
+    "Continue where I left off": [
+        null,
+        ""
+    ],
+    "Start over": [
```

Comment:
Does the welcome back screen not have translations? This screen can be shown to a user after they've selected a language